packages/biome-config/biome.jsonc

```
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://biomejs.dev/schemas/2.1.1/schema.json",
```

should we do something like this[^1] instead so our versions stay in sync?

```suggestion
  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json"
```

[^1]: [source](https://biomejs.dev/reference/configuration/#schema)